src/components/charts/charts.css

```diff
@@ -342,4 +342,8 @@
 
 .chart-group-deployment__delete-app:hover .stroke-color {
     stroke: var(--R500);
+}
+
+.flex {
```

Comment:
Is it required? seems unused

Reply:
"form-error" class css overiding and making it "display: block;" but i want display as flex so i written it inside chart.css